openstef/tasks/create_components_forecast.py

```diff
@@ -92,17 +93,15 @@ def create_components_forecast_task(pj: PredictionJobDataClass, context: TaskCon
         datetime_end=datetime_end,
     )
 
-    # Get splitting coeficients
-    split_coefs = context.database.get_energy_split_coefs(pj)
-
-    if len(split_coefs) == 0:
-        logger.warning(f"No Coefs found. Skipping pid", pid=pj["id"])
-        return
-
     # Make forecast for the demand, wind and pv components
-    forecasts = create_components_forecast_pipeline(
-        pj, input_data, weather_data, split_coefs
-    )
+    forecasts = create_components_forecast_pipeline(pj, input_data, weather_data)
+
+    if forecasts.index.max() < datetime.utcnow().replace(tzinfo=pytz.utc) + timedelta(
```

Comment:
2 days is too long I think.
We retrieve Harm_Arome data for more or less 48 hours ahead, but if it is only available up to 36 hours ahead every now and again, that is to be expected

Reply:
Good tip!, I set it to 30 hours now.